### ISSUE TITLE
implement "Algorithms" page under user menu

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-client#104]
BTW what does this page means (needs to be documented in the wiki at least)?

![image](https://cloud.githubusercontent.com/assets/180794/5599692/0d9cb8e4-92cc-11e4-84aa-222d4c7075c7.png)

**Comment by pietrosperoni** [comment]
@giovanisp The algorithms page is just a page for administrators to see the inns and outs of the algorithm, and see if the pareto front is correct. For now it should be commented out.

So, @derekpa could you please comment out this page for non administrators?

**Comment by giovanisp** [comment]
moved to next release as this needs roles/ACL ("administrators", permissions...).

**Comment by derekpa** [comment]
Yes, we can just have a simple admin role for now as in vilfredo 1.

Are you still using the algorithms page pietro?

**Comment by pietrosperoni** [comment]
No, thanks. Not at the moment. But there are some new tests we could do soon, so I would not delete it completely.
